client/src/modules/stock/lots-duplicates/duplicate-lots.html

```diff
@@ -0,0 +1,59 @@
+<div class="flex-header">
```

Comment:
A meta comment - I find it a little strange this file path is `lots-duplicates/duplicate-lots.html`.  Is a reason why the folder name and the file name are different?

Reply:
I wanted this to sort next to the 'lots' file folder uner 'stock', hence the name. I plan to rename the files to 'lots-duplicates.*' to match the directory name. 'duplicate-lots' would be more readable, but would make it a bit harder to find with the file explorer.

Reply:
Files renamed.